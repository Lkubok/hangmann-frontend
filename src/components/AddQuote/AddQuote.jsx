import React, { Component } from "react";
import QuoteFormik from "../QuoteForm";

export class AddQuote extends Component {
  render() {
    return <QuoteFormik />;
  }
}
export default AddQuote;
