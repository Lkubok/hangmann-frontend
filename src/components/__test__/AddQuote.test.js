import React from "react";
import Enzyme, { shallow } from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import { AddQuote } from "../AddQuote/AddQuote";
Enzyme.configure({ adapter: new Adapter() });

describe("AddQuote component", () => {
  beforeEach(() => {});
  it("Should match snapshot", () => {
    const wrapper = shallow(<AddQuote />);
    expect(wrapper).toMatchSnapshot();
  });
});
