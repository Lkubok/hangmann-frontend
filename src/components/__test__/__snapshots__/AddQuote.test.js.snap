// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddQuote component Should match snapshot 1`] = `<withRouter(Connect(WithFormik(QuoteForm))) />`;
